ansible_collections/arista/avd/roles/eos_designs/python_modules/metadata/cv_pathfinder.py

```diff
@@ -199,3 +199,12 @@ def _get_vni_for_vrf_name(self: AvdStructuredConfigMetadata, vrf_name: str):
             raise AristaAvdError(f"Unable to find the WAN VNI for VRF {vrf_name} during generation of cv_pathfinder metadata.")
 
         return wan_vni
+
+    def _metadata_internet_exit_policies(self: AvdStructuredConfigMetadata):
```

Comment:
This changes the order of inserting the metadata in structured_config - not necessarily bad but still to consider?

Reply:
Structured config is always ok to reorder.